misc/python/materialize/output_consistency/ignore_filter/inconsistency_ignore_filter.py

```diff
@@ -227,12 +235,26 @@ def shall_ignore_error(self, error: ValidationError) -> IgnoreVerdict:
                 EvaluationStrategyKey.CONSTANT_FOLDING
             ].successful
 
-            if (
-                error.query_execution.query_template.contains_aggregations
-                and not dfr_successful
-                and ctf_successful
-            ):
+            if not dfr_successful and ctf_successful:
                 # see https://github.com/MaterializeInc/materialize/issues/19662
                 return YesIgnore("#19662")
 
+        if error.error_type == ValidationErrorType.ERROR_MISMATCH:
+            query_template = error.query_execution.query_template
+
+            FUNCTIONS_TAKING_SHORTCUTS = {"count"}
```

Comment:
As of now, `count` is known to cause the shortcuts. If possible, I would like to restrict the ignore logic to a known set of problematic functions rather than completely giving up error message comparisons.